### PR TITLE
feat: add airflow

### PR DESCRIPTION
**Pull request by sidohaakma**
#### Checklist
- [x] Functionality works & meets specifications (tested on rancher)
- [x] Templates reviewed
- [x] Added values to questions
- [x] Bumped the chart version
- [x] Updated appVersion (if needed)

### CODE REVIEW

File: charts/molgenis-airflow/values.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+airflow:
+
+  airflow:
+    config:
+      AIRFLOW__WEBSERVER__BASE_URL: http://airflow.edge.molgenis.org/airflow
+      AIRFLOW__CELERY__FLOWER_URL_PREFIX: /airflow/flower
+
```
Comment (connoratrug) [inline charts/molgenis-airflow/values.yaml:7]: suggest to add admin pr secret

File: charts/molgenis-airflow/values.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  dags:
+    gitSync:
+      enabled: true
+      repo: https://github.com/sidohaakma/molgenis-tools-airflow
```
Comment (connoratrug) [inline charts/molgenis-airflow/values.yaml:11]: suggest to move to molgenis repo ( even for poc)

File: charts/molgenis-airflow/values.yaml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+  airflow:
+    config:
+      AIRFLOW__WEBSERVER__BASE_URL: https://airflow.molgenis.org
+      AIRFLOW__CELERY__FLOWER_URL_PREFIX: /flower
+    extraEnv:
+      - name: FUSION_AUTH_CLIENT_ID
+        value: 2e1d179b-7ec1-4371-8543-74cd18b3acf3
+      - name: FUSION_AUTH_CLIENT_SECRET
+        value: b_YAoAOxld32OtlFG20g1xmkVOWkSn5kCdzI-HUDJKs
```
Comment (connoratrug) [inline charts/molgenis-airflow/values.yaml:11]: moet deze hier al ingevuld staan ?

**Comment by sidohaakma** [comment]
[#8184 Airflow afmaken](https://trello.com/c/ijpvvWTa/249-8184-airflow-afmaken)

**Review by connoratrug — DISMISSED** [review]

**Review by connoratrug — DISMISSED** [review]

**Pull request merged**